src/librustc_resolve/lib.rs

```diff
@@ -934,12 +933,10 @@ pub struct Resolver<'a> {
     multi_segment_macro_resolutions: Vec<(Vec<Segment>, Span, MacroKind, ParentScope<'a>,
                                           Option<Res>)>,
     builtin_attrs: Vec<(Ident, ParentScope<'a>)>,
-    /// Some built-in derives mark items they are applied to so they are treated specially later.
+    /// `derive(Copy)` marks items they are applied to so they are treated specially later.
     /// Derive macros cannot modify the item themselves and have to store the markers in the global
     /// context, so they attach the markers to derive container IDs using this resolver table.
-    /// FIXME: Find a way for `PartialEq` and `Eq` to emulate `#[structural_match]`
-    /// by marking the produced impls rather than the original items.
-    special_derives: FxHashMap<ExpnId, SpecialDerives>,
+    copy_derives: FxHashSet<ExpnId>,
```

Comment:
These are IDs of `#[derive(...)]` containers having `Copy` inside them rather than IDs of `Copy` derives themselves, so it would be better to rename this into something less misleading.
`have_copy_derives`?
`containers_having_copy_derive`?

Reply:
going to go with `containers_deriving_copy`